app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from typing import Dict, Any
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: Dict[str, Any]) -> str | Exception:

```

Comment:
According to the checklist, you should use type annotations. The function return type should be `Union[str, Exception]` instead of `str | Exception`.